### PR TITLE
feat: revamp grant creation flow

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- Revamp Grant Creation
- Hide Grant Type and default it to gr12

Note: Ensure gr12 is created on backend  

##### Testing
Untitled

### CODE REVIEW

**Review by chibie — COMMENTED** [review]
can we have screenshots (desktop+responsive) or a vid?

**Comment by gdixon** [comment]
@ksolo - could you have a look at the cyprus tests on this one please?

File: cypress/integration/grants/test_grant_creation.js [hunk 1]
```diff
@@ -46,13 +46,17 @@ describe('Creating a new grant', () => {
...
         cy.get('input[name=eth_payout_address]').type('0xd08Fe0c97c80491C6ee696Ee8151bc6E57d1Bf1d');
-        cy.get('input[placeholder="Yes/No"]').click();
+
+        cy.get('input[placeholder="Has this project received external funding?"]').click();
         cy.contains('No, this project has not raised external funding.').click();
 
-        cy.get('input[placeholder="Pick a category"]').click();
-        cy.contains('Community').click();
+        // cy.get('input[placeholder="Pick a category"]').click();
+        // cy.contains('Community').click();
```
Comment (gdixon) [inline cypress/integration/grants/test_grant_creation.js:58]: Do we need to keep these?

**Review by gdixon — APPROVED** [review]
LGTM! 🚀

File: cypress/integration/grants/test_grant_creation.js (hunk 1 above)
Comment (thelostone-mc) [reply cypress/integration/grants/test_grant_creation.js:58]:
```suggestion
       
```

File: cypress/integration/grants/test_grant_creation.js [hunk 2]
```diff
@@ -46,13 +46,17 @@ describe('Creating a new grant', () => {
...
+
         cy.get('input[name=eth_payout_address]').type('0xd08Fe0c97c80491C6ee696Ee8151bc6E57d1Bf1d');
-        cy.get('input[placeholder="Yes/No"]').click();
+
+        cy.get('input[placeholder="Has this project received external funding?"]').click();
         cy.contains('No, this project has not raised external funding.').click();
 
-        cy.get('input[placeholder="Pick a category"]').click();
-        cy.contains('Community').click();
+        // cy.get('input[placeholder="Pick a category"]').click();
```
Comment (thelostone-mc) [inline cypress/integration/grants/test_grant_creation.js:57]:
```suggestion
```

**Comment by frankchen07** [comment]
functionality check - LGTM!

**Comment by frankchen07** [comment]
- [ ] can we alpha sort tags?

**Pull request merged**